src/app/views/main/main.component.html

```diff
@@ -13,13 +13,18 @@
         <mat-drawer-content class="contents">
           <div *ngIf="!(navigation.isHomeActive | async)" class="top-bar">
             <div class="search-criteria">{{ search.searchCriteria | async }}</div>
+            <div class="download-button">
+              <a href="https://sampledata.hubmapconsortium.org/home" target="_blank">
+                <mat-icon class="download-icon">cloud_download</mat-icon>
```

Comment:
Classes should be 'icon font-icon' for consistency and clarity (look at other mat-icon usage in other files)